integrations/gitlab-v2/gitlab/clients/gitlab_client.py

```diff
@@ -123,6 +100,81 @@ async def get_groups_resource(
             if batch:
                 yield batch
 
+    async def get_file_content(
+        self, project_id: str, file_path: str, ref: str = "main"
+    ) -> Optional[str]:
+        return await self.rest.get_file_content(project_id, file_path, ref)
+
```

Comment:
I think ref should be default 🤔 is this possible? as not all are calling their main branch `main`

Reply:
I do see that we are passing to it always a ref, so maybe worth just removing the `=main`